core/src/main/resources/lib/layout/html.jelly

```diff
@@ -0,0 +1,177 @@
+<!--
```

Comment:
Seems like a lot of this is copied from `layout.jelly` that should have been reused instead, or this could just have been a `decorate="false"` option to `l:layout`?